Hymns/Common/SearchBar.swift

```diff
@@ -27,6 +24,11 @@ struct SearchBar: View {
                         self.searchText = ""
                     }
                 }, label: {Image(systemName: "xmark.circle.fill").opacity(self.searchText.isEmpty ? 0.0 : 1.0)})
+            }.onTapGesture {
+                if !self.searchActive {
+                    self.searchActive = true
```

Comment:
What happens to the cancel button then if you do this?

Reply:
It's fine. Their are two HStacks. Cancel button is part of the outter HStack which is not effected by this since this is on the inner stack